packages/api/config.js

```diff
@@ -33,7 +33,7 @@ nconf
     parseValues: true,
     transform: obj => {
       // remove the "SPASHIP_" prefix from environment variables
-      obj.key = obj.key.replace(/^spaship_/, "");
+      obj.key = obj.key.replace(/^spaship_/i, "").replace(/api_/i, "");
```

Comment:
Just in case "api_" exists inside of a property name at some point.

```suggestion
      obj.key = obj.key.replace(/^spaship_/i, "").replace(/^api_/i, "");
```